### ISSUE TITLE
Finished?

### ISSUE THREAD

**Issue by rafwill** [blotspot/garmin-watchface-protomolecule#1]
Is this watchface finished?? I don't see it in Garmin Connect page
Thanks four protomolecule. Is great!!

**Comment by blotspot** [comment]
Hi there,

thanks for your interest. I would not consider it a 100% finished yet. The functionality is pretty solid but I haven't fully optimized it for Watches in the 240px to 280px range. 
Furthermore the design pictures I created roughly 2 years ago (in photoshop) are not entirely up to date. The central clock and date bit is untouched but the icons have changed since they just didn't work for me on my tiny vivoactive 4s. I also currently nixed  the numbers on the primary indicators cause it felt a bit too cramped and you already have the indicators to tell you where you're at. I might add them again as a configurable option though.
I also implemented a second design. The clock and the three data fields under it are unchanged but the indicators are different and allow for more data fields to be shown.

tl;dr: Yes, but it needs a bit of polishing and I don't want to deceive anyone with shiny pictures are are a bit outdated.

**Comment by blotspot** [comment]
Watch Face can now be found here: https://apps.garmin.com/en-US/apps/d9f524cc-b8e3-41df-b0d3-967b1743d165
